src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec

```diff
@@ -16,6 +16,7 @@
   </metadata>
   <files>
     <file src="bin\Release\FakeItEasy.Analyzer.dll" target="analyzers\dotnet\cs" />
+    <file src="bin\Release\FakeItEasy.Analyzer.pdb" target="analyzers\dotnet\cs" />

```

Comment:
It's questionable whether this will be of much use, but I don't think it hurts to include it. I guess there may be a scenario where someone one wants to troubleshoot a problem in the analyzer by invoking it directly and stepping through, e.g. I potentially could have done this when investigating #731.